### PR TITLE
tiflash: dt_enable_read_thread depcrecated

### PR DESCRIPTION
**Pull request by JinheLin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by joey-yez — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -188,8 +188,7 @@ delta_index_cache_size = 0
     ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
     dt_page_gc_threshold = 0.5
 
-    ## 从 v6.2.0 引入，使用线程池处理存储引擎的读请求。默认为 false。
-    ## 警告：目前是实验性功能，不建议在生产环境中使用。
+    ## 该参数从 v6.3.0 开始废弃，默认开启此功能且不能关闭。
     # dt_enable_read_thread = false
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:192]: 如果默认开启，false 需要改成 true 吗

Comment (hfxsd) [inline tiflash/tiflash-configuration.md:192]:
```suggestion
    # dt_enable_read_thread = true
```

Comment (hfxsd) [reply tiflash/tiflash-configuration.md]: 我也问过同样的问题。现在改成啥都不会起作用，这个参数形同虚设。我改成 true 吧，更符合逻辑。

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -188,9 +188,8 @@ delta_index_cache_size = 0
     ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
     dt_page_gc_threshold = 0.5
 
-    ## 从 v6.2.0 引入，使用线程池处理存储引擎的读请求。默认为 false。
-    ## 警告：目前是实验性功能，不建议在生产环境中使用。
-    # dt_enable_read_thread = false
+    ## 该参数从 v6.3.0 开始废弃，默认开启此功能且不能关闭。
```
Comment (lilinghai) [inline tiflash/tiflash-configuration.md:191]: 既然要废弃了，为啥还要在文档中列出来?

Comment (flowbehappy) [reply tiflash/tiflash-configuration.md:191]: I think we can just remove this setting.

**Review by flowbehappy — APPROVED** [review]

**Review by lilinghai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**